src/btree/bt_compact.c

```diff
@@ -391,6 +391,22 @@ __wt_compact(WT_SESSION_IMPL *session)
             WT_WITH_PAGE_INDEX(session, ret = __compact_walk_internal(session, ref));
         }
 
+        if (ret == EBUSY) {
+            WT_STAT_CONN_INCR(session, session_table_compact_conflicting_checkpoint);
+
+            __wt_verbose_level(session, WT_VERB_COMPACT, WT_VERBOSE_DEBUG_1,
+              "The compaction of the data handle %s returned EBUSY due to an in-progress "
+              "conflicting checkpoint.%s",
+              session->dhandle->name,
+              session == S2C(session)->background_compact.session ?
+                "" :
+                " Compaction of this data handle will resume after checkpoint completes.");
```

Comment:
If we decide to keep the same behaviour for bg and fg compaction, we should update this log.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/10929/commits/c519e4463d583d5bb9d5a26a7778c254f881d079